nxdrive/gui/folders_dialog.py

```diff
@@ -275,6 +277,13 @@ def __init__(
         self.tree_view.setContextMenuPolicy(Qt.CustomContextMenu)
         self.tree_view.customContextMenuRequested.connect(self.open_menu)
 
+    def keyPressEvent(self, event: QKeyEvent) -> None:
```

Comment:
Add a comment with the ticket number. This modification is specific to one client and we should have a trace of why we changed the behaviour.

Reply:
👍